packages/dynamics-lib/src/entities/__tests__/pocl-data-validaton-error.entity.spec.js

```diff
@@ -0,0 +1,157 @@
+import { PoclDataValidationError } from '../pocl-data-validation-error.entity.js'
+import { retrieveGlobalOptionSets } from '../..'
+
+let optionSetData
+describe('pocl staging exception entity', () => {
+  beforeAll(async () => {
+    optionSetData = await retrieveGlobalOptionSets().cached()
```

Comment:
I'm not a big fan of `let` declarations - it's probably better to get it for each test rather than getting it once and re-using it. Then the test is self contained.